packages/lib-classifier/src/components/Classifier/components/FieldGuide/components/FieldGuide/components/FieldGuideItem/FieldGuideItem.js

```diff
@@ -27,17 +26,24 @@ const StyledButton = styled(Button)`
       }
   `}
 `
+
+const StickyTitle = styled(Box)`
+  position: sticky;
+  top: -1px;
+  z-index: 999;
+`
+
 const markdownTitleComponent = {
-  h3: (nodeProps) => <SpacedHeading level='3' margin='none'>{nodeProps.children}</SpacedHeading>
+  h3: (nodeProps) => <SpacedHeading level={3} margin='none'>{nodeProps.children}</SpacedHeading>
```

Comment:
The change from string to number in this file was prompted by a dev console warning. SpacedHeading component expects a number for `level`.